wicketd/src/http_entrypoints.rs

```diff
@@ -59,21 +58,26 @@ async fn get_inventory(
     }
 }
 
-/// An endpoint used to upload artifacts to the server.
+/// Upload TUF repositories to the server.
+///
+/// At any given time, wicketd will keep at most one repository in memory. Any
+/// previously-uploaded repositories will be discarded.
 #[endpoint {
     method = PUT,
-    path = "/artifacts/{name}/{version}",
+    path = "/repositories/{name}/{version}",
```

Comment:
I'm not sure what the name/version are buying us now that we hold at most one repository. If we need to identify it, should that identification come from metadata within the repository itself instead of args from the uploader?

Reply:
Good question! I'd put it in for logging purposes but we don't need it. Agreed that having a file with this would make sense, though we don't currently have a place to store that. (@iliana maybe in `artifacts.json`?)

I guess we should also change the name to `repository` rather than `repositories` in that case.

Reply:
Hmm, that still has the problem that the name/version here are effectively arbitrary and coming from an unverifiable source outside the rack. How much logging detail do we need if someone uploads a bad repository?

Reply:
I guess we don't really, no. I've updated the PR to remove the name and version.